pkg/gardenlet/controller/add.go

```diff
@@ -85,6 +88,14 @@ func AddToManager(
 		return fmt.Errorf("failed adding ControllerInstallation controller: %w", err)
 	}
 
+	if err := (&managedseed.Reconciler{
+		Config:          *cfg.Controllers.ManagedSeed,
+		ChartsPath:      filepath.Join(charts.Path, "gardener", "gardenlet"),
+		GardenNamespace: gardenNamespace.Name,
```

Comment:
```suggestion
```
You already set this inside `AddToManager`.
Maybe change `NetworkPolicy` also along the way to be consistent with other controllers.

Reply:
Here I need the garden namespace to be the one created for this test and not the default `garden` namespace. Inside `AddToManager`, `GardenNamespace` is set to `v1beta1constants.GardenNamespace`.

Reply:
> Inside AddToManager, GardenNamespace is set to v1beta1constants.GardenNamespace.

Only if it is empty.